perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -727,38 +727,42 @@ sub _nearby_json :Private {
         # This is for the list template, this is a list on that page.
         $c->stash->{page} = 'report';
 
-        my $dist = $self->_find_distance($c, $params);
-        $params->{distance} = $dist / 1000 unless $params->{distance}; # DB measures in km
+        if (my $dist = $self->_find_distance($c, $params)) { # distance of 0 means we can skip lookup entirely
```

Comment:
Is fine, but could be done as an early return if not present, then don't need to indent everything further